cmd/control-heal-main.go

```diff
@@ -75,14 +75,27 @@ func healControl(ctx *cli.Context) {
 	parsedURL, err := url.Parse(ctx.Args()[0])
 	fatalIf(err, "Unable to parse URL")
 
+	client, err := rpc.DialHTTPPath("tcp", parsedURL.Host, path.Join(reservedBucket, controlPath))
```

Comment:
Can you use the 'newAuthClient' style done with the latest HEAD on 'distributed' branch.